### PR TITLE
Create LICENSE.md

### LINKED ISSUE #2660: Add a lisence to the project
̶#̶#̶#̶ ̶F̶e̶a̶t̶u̶r̶e̶ ̶r̶e̶q̶u̶e̶s̶t̶

̶*̶*̶F̶e̶a̶t̶u̶r̶e̶ ̶d̶e̶s̶c̶r̶i̶p̶t̶i̶o̶n̶*̶*̶
̶A̶d̶d̶ ̶a̶ ̶l̶i̶c̶e̶n̶s̶e̶ ̶t̶o̶ ̶t̶h̶e̶ ̶p̶r̶o̶j̶e̶c̶t̶.̶ ̶A̶ ̶l̶i̶c̶e̶n̶s̶e̶ ̶i̶s̶ ̶a̶b̶s̶o̶l̶u̶t̶e̶l̶y̶ ̶n̶e̶c̶e̶s̶s̶a̶r̶y̶ ̶f̶o̶r̶ ̶a̶n̶y̶ ̶g̶i̶t̶h̶u̶b̶ ̶p̶r̶o̶j̶e̶c̶t̶.̶

̶*̶*̶H̶o̶w̶ ̶t̶h̶e̶ ̶f̶e̶a̶t̶u̶r̶e̶ ̶i̶s̶ ̶u̶s̶e̶f̶u̶l̶*̶*̶
̶A̶ ̶l̶i̶c̶e̶n̶s̶e̶ ̶w̶o̶u̶l̶d̶ ̶a̶l̶l̶o̶w̶ ̶p̶e̶o̶p̶l̶e̶ ̶t̶o̶ ̶l̶e̶g̶a̶l̶l̶y̶ ̶c̶o̶n̶t̶r̶i̶b̶u̶t̶e̶ ̶t̶o̶ ̶t̶h̶e̶ ̶p̶r̶o̶j̶e̶c̶t̶.̶ ̶C̶u̶r̶r̶e̶n̶t̶l̶y̶ ̶i̶t̶ ̶i̶s̶ ̶i̶l̶l̶e̶g̶a̶l̶ ̶t̶o̶ ̶s̶u̶b̶m̶i̶t̶ ̶p̶u̶l̶l̶ ̶r̶e̶q̶u̶e̶s̶t̶s̶ ̶t̶o̶ ̶t̶h̶e̶ ̶p̶l̶u̶g̶i̶n̶ ̶a̶n̶d̶ ̶c̶h̶a̶n̶g̶e̶ ̶t̶h̶e̶ ̶s̶o̶u̶r̶c̶e̶ ̶c̶o̶d̶e̶.̶ ̶F̶o̶r̶ ̶m̶o̶r̶e̶ ̶i̶n̶f̶o̶r̶m̶a̶t̶i̶o̶n̶ ̶s̶e̶e̶:̶ ̶h̶t̶t̶p̶s̶:̶/̶/̶c̶h̶o̶o̶s̶e̶a̶l̶i̶c̶e̶n̶s̶e̶.̶c̶o̶m̶/̶n̶o̶-̶p̶e̶r̶m̶i̶s̶s̶i̶o̶n̶/̶

̶Y̶o̶u̶ ̶a̶d̶d̶ ̶t̶h̶e̶ ̶l̶i̶c̶e̶n̶s̶e̶ ̶b̶y̶ ̶a̶d̶d̶i̶n̶g̶ ̶a̶ ̶f̶i̶l̶e̶ ̶n̶a̶m̶e̶d̶ ̶"̶L̶I̶S̶E̶N̶C̶E̶"̶ ̶t̶o̶ ̶t̶h̶e̶ ̶r̶e̶p̶o̶.̶ ̶(̶D̶o̶n̶'̶t̶ ̶a̶d̶d̶ ̶a̶n̶y̶ ̶f̶i̶l̶e̶ ̶e̶x̶t̶e̶n̶s̶i̶o̶n̶ ̶l̶i̶k̶e̶ ̶.̶t̶x̶t̶ ̶t̶o̶ ̶i̶t̶.̶ ̶J̶u̶s̶t̶ ̶n̶a̶m̶e̶ ̶i̶t̶ ̶L̶I̶C̶E̶N̶S̶E̶)̶

̶G̶o̶o̶d̶ ̶l̶i̶s̶e̶n̶c̶e̶s̶ ̶a̶r̶e̶ ̶G̶n̶u̶ ̶G̶P̶L̶V̶3̶ ̶(̶h̶t̶t̶p̶s̶:̶/̶/̶w̶w̶w̶.̶g̶n̶u̶.̶o̶r̶g̶/̶l̶i̶c̶e̶n̶s̶e̶s̶/̶g̶p̶l̶-̶3̶.̶0̶.̶t̶x̶t̶)̶,̶ ̶M̶I̶T̶ ̶L̶i̶c̶e̶n̶s̶e̶ ̶(̶h̶t̶t̶p̶s̶:̶/̶/̶c̶h̶o̶o̶s̶e̶a̶l̶i̶c̶e̶n̶s̶e̶.̶c̶o̶m̶/̶l̶i̶c̶e̶n̶s̶e̶s̶/̶m̶i̶t̶/̶)̶,̶ ̶G̶n̶u̶ ̶L̶G̶P̶L̶V̶3̶ ̶(̶h̶t̶t̶p̶s̶:̶/̶/̶w̶w̶w̶.̶g̶n̶u̶.̶o̶r̶g̶/̶l̶i̶c̶e̶n̶s̶e̶s̶/̶l̶g̶p̶l̶-̶3̶.̶0̶.̶t̶x̶t̶)̶,̶ ̶A̶p̶a̶c̶h̶e̶ ̶2̶.̶0̶ ̶L̶i̶c̶e̶n̶s̶e̶ ̶(̶h̶t̶t̶p̶s̶:̶/̶/̶w̶w̶w̶.̶a̶p̶a̶c̶h̶e̶.̶o̶r̶g̶/̶l̶i̶c̶e̶n̶s̶e̶s̶/̶L̶I̶C̶E̶N̶S̶E̶-̶2̶.̶0̶.̶t̶x̶t̶)̶

̶M̶o̶r̶e̶ ̶i̶n̶f̶o̶r̶m̶a̶t̶i̶o̶n̶ ̶a̶b̶o̶u̶t̶ ̶l̶i̶c̶e̶n̶s̶i̶n̶g̶ ̶s̶t̶u̶f̶f̶ ̶h̶t̶t̶p̶s̶:̶/̶/̶c̶h̶o̶o̶s̶e̶a̶l̶i̶c̶e̶n̶s̶e̶.̶c̶o̶m̶/̶

̶A̶l̶s̶o̶ ̶i̶t̶ ̶w̶o̶u̶l̶d̶ ̶b̶e̶ ̶u̶s̶e̶f̶u̶l̶ ̶i̶f̶ ̶y̶o̶u̶ ̶a̶d̶d̶e̶d̶ ̶t̶h̶e̶ ̶l̶i̶c̶e̶n̶s̶e̶ ̶t̶o̶ ̶a̶l̶l̶ ̶o̶t̶h̶e̶r̶ ̶M̶u̶l̶t̶i̶v̶e̶r̶s̶e̶ ̶r̶e̶p̶o̶s̶.̶

Nvm I thought the project didn't have a license at all. I will make a pull request to add the license to the project. I will also make the pull request to all the other repos which only have it in the readme.

However some old Multiverse projects (https://github.com/Multiverse/Multiverse-Adventure, https://github.com/Multiverse/Multiverse-EssentialsSupport, https://github.com/Multiverse/multiverse.github.com, https://github.com/Multiverse/Multiverse-Layers and https://github.com/Multiverse/Baxter) do not have a license. Not sure if they are used or not but I suggest some sort of license should be added to them.

### PR DESCRIPTION
**Pull request by ccuser44**
Fixes #2660

Added the license from README.md to LICENSE.md for clarity

Had to make it .md so the license would properly wrap.

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Copyright (c) 2011, The Multiverse Team All rights reserved.
```
Comment (nicegamer7) [inline LICENSE.md:1]: Should probably change this to match the current year, both here and in the README.

**Review by nicegamer7 — COMMENTED** [review]
Looks good, GitHub correctly identifies the license as the BSD-3-Clause License.

File: LICENSE.md (hunk 1 above)
Comment (dumptruckman) [reply LICENSE.md:1]: Agreed.

Comment (dumptruckman) [reply LICENSE.md:1]: Actually, probably remove the license from the README.

Comment (benwoo1110) [reply LICENSE.md:1]: Then readme will be basically empty haha, I think time for #2519 as well

**Review by dumptruckman — APPROVED** [review]
Looks good enough. We can change the copyright date later.

**Comment by FernFerret** [comment]
As long as Multiverse Stays MIT/BSD that's all I really want :) Pretty funny that myself (nor anyone else) has changed the date in 10 years. Happy 10+ yr anniversary all!

**Pull request merged**